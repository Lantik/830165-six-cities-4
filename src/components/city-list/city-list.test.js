import React from 'react';
import renderer from 'react-test-renderer';
import CityList from './city-list.jsx';

const mock = {
  cities: [`Amsterdam`, `Dusseldorf`, `London`, `Paris`]
};

it(`<CityList/> renders correctly`, () => {
  const {cities} = mock;

  const tree = renderer
    .create(<CityList
      cities={cities}
      activeCity={`Amsterdam`}
      onCityTitleClick={() => {}}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
