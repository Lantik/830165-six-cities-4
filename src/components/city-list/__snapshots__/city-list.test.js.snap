// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CityList/> renders correctly 1`] = `
<ul
  className="locations__list tabs__list"
>
  <li
    className="locations__item"
    onClick={[Function]}
  >
    <a
      className="locations__item-link tabs__item tabs__item--active"
      href="#"
    >
      <span>
        Amsterdam
      </span>
    </a>
  </li>
  <li
    className="locations__item"
    onClick={[Function]}
  >
    <a
      className="locations__item-link tabs__item "
      href="#"
    >
      <span>
        Dusseldorf
      </span>
    </a>
  </li>
  <li
    className="locations__item"
    onClick={[Function]}
  >
    <a
      className="locations__item-link tabs__item "
      href="#"
    >
      <span>
        London
      </span>
    </a>
  </li>
  <li
    className="locations__item"
    onClick={[Function]}
  >
    <a
      className="locations__item-link tabs__item "
      href="#"
    >
      <span>
        Paris
      </span>
    </a>
  </li>
</ul>
`;
